libssh2.hpp

```diff
@@ -324,6 +331,40 @@ namespace libssh2
             channel* c = new channel(_chan);
             return c;
         }
+
+
+        channel* open_scp_channel(std::string path, FILE *local)
+        {
+            struct stat fileinfo;
+               
+            stat("./test.txt", &fileinfo);
```

Comment:
Why refer to a `test.txt` file? Shouldn't this be the `path` variable given as argument?

Reply:
Sorry, please forget that pull request, I mixed it up with some tests from my side, I will resend an Request when I have time.